static/templates/navbar.hbs

```diff
@@ -45,6 +45,17 @@
             </div>
         </div>
         <div class="column-right">
+            <div class="spectator_login_buttons only-visible-to-spectators hide-xl">
```

Comment:
Should this be `only-visible-for-spectators`? Also, can we skip the `only-visible-for-spectators` on the divider inside this if we fix that typo?

Reply:
Ok, fixed!